docs/modules/reference/pages/ticketing/ticketer/jira.adoc

```diff
@@ -7,24 +7,31 @@ The Jira ticketing plugin creates JIRA issues in response to {page-component-tit
 [[ref-ticketing-jira-setup]]
 == Setup
 
+IMPORTANT: The use of the JIRA ticketer requires the installation of the `opennms-plugin-ticketer-jira` package before the plugin will start.
+
+****
 To install the opennms-plugin-ticketer-jira package:
 
 * For RPM-based distributions:
 ** `yum install opennms-plugin-ticketer-jira`
 * For Debian-based distributions:
 ** `apt-get install opennms-plugin-ticketer-jira`
+****
 
-The Jira ticketing plugin and its dependencies are not part of the core packages.
+. Once installed, set the following property in `$OPENNMS_HOME/etc/opennms.properties.d/ticketer.properties` to enable the plugin:
 
-To enable the plugin, start by setting following property in `$OPENNMS_HOME/etc/opennms.properties`:
+ opennms.ticketer.plugin=org.opennms.netmgt.ticketd.OSGiBasedTicketerPlugin
 
-[source, properties]
-----
-opennms.ticketer.plugin=org.opennms.netmgt.ticketd.OSGiBasedTicketerPlugin
-----
+. Configure the plugin options by setting desired options in `$OPENNMS_HOME/etc/jira.properties`.
+See the table below for available options.
+. To make the ticketer system start automatically in the future, add `jira-troubleticketer` to the `featuresBoot` property in the `$OPENNMS_HOME/etc/org.apache.karaf.features.cfg`.
```

Comment:
echo "jira-troubleticketer" >> $OPENNMS_HOME/etc/featuresBoot.d/ticketer.boot

Bad practice to append things to org.apache.karaf.features.cfg, as it can be overwritten by fix-karaf-setup.sh, or by upgrade.